cross/urbackup/Makefile

```diff
@@ -0,0 +1,30 @@
+PKG_NAME = urbackup-server
+PKG_VERS = 2.4.12
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://hndl.urbackup.org/Server/$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS  = cross/curl cross/openssl cross/zlib
+
+HOMEPAGE = https://www.urbackup.org/
+COMMENT  = UrBackup is an easy to setup open source client/server backup system, that through a combination of image and file backups accomplishes both data safety and a fast restoration time.
+LICENSE  = GNU GPL
+
+CONFIGURE_ARGS = --enable-packaging --enable-embedded-cryptopp --with-zlib=$(STAGING_INSTALL_PREFIX) #--without-crypto --without-zlib --without-mail 
+GNU_CONFIGURE = 1
+ADDITIONAL_CXXFLAGS = -Os
+ADDITIONAL_CFLAGS = -Os
+
+
+INSTALL_TARGET = myInstall
```

Comment:
If this target is really necessary, please rename to match our naming policy: `urbackup_install`

Reply:
changed